pac4j-couch/pom.xml

```diff
@@ -26,6 +26,11 @@
 			<artifactId>org.ektorp</artifactId>
 			<version>${ektorp.version}</version>
 		</dependency>
+        <dependency>
```

Comment:
indentation is funny here and in other pom.xml... I think it is because it was using tabs before, but now we have an editorconfig file that enforces 2-spaces indentation... maybe it would be good to reindent all pom at one point (or change the editorconfig setting to 1-tab indentation, but it's not the best usually for xml-based language, because it's not rendered identically everywhere)